nixos/modules/services/networking/go-shadowsocks2.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.go-shadowsocks2.server;
+in {
+  options.services.go-shadowsocks2.server = {
+    enable = mkEnableOption "go-shadowsocks2 server";
+
+    listenAddress = mkOption {
+      type = types.str;
+      description = "Server listen address or URL";
```

Comment:
No default?

Reply:
The listen address is a [special URI](https://github.com/shadowsocks/go-shadowsocks2#server), which also contains login credentials. Thus I do not consider a default to be meaningful.

Reply:
I added an example based on the project's readme. However, there is no sane default for this field.